physics/sfc_drv_ruc.F90

```diff
@@ -260,7 +258,7 @@ subroutine lsm_ruc_run                                            & ! inputs
      &       srflag, snoalb, isot, ivegsrc, fice, smcwlt2, smcref2,     &
      ! --- constants
      &       con_cp, con_rd, con_rv, con_g, con_pi, con_hvap,           &
-     &       con_fvirt,                                                 &
+     &       con_fvirt, min_seaice, con_tice, rhowater,                 &
```

Comment:
You could just call them cimin and rhoh2o here and save to local variables. Since they are scalars, it doesn't really matter, though.

Reply:
Could min_seaice be changed in the input.nml? The 15% value they use is higher that what we use in WRF.